invenio_records_resources/errors.py

```diff
@@ -10,10 +10,40 @@
 
 import json
 
+from flask import abort, current_app, jsonify, make_response, request, url_for
```

Comment:
Remove unused imports

```suggestion
from flask import jsonify, make_response, request, url_for
```